### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
Milestones 💪
- [ ] Add navigation bar pages using React router
 - [ ] Create a directory of components that contains
 - [ ] Books component as a default view and displays a list of book items with their respective action buttons {comments, remove 
         and edit}
 - [ ] Book component which displays the newly added book
 - [ ] Add a new book component which able to add a new item.
 - [ ] Categories component only displays a "Check status" button.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  BrowserRouter, Route, Link, Routes,
+} from 'react-router-dom';
+import Categories from './components/catagories';
+import Books from './components/books';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Link to="/">Books</Link>
+      <Link to="/Categories">Categories</Link>
```
Comment (codepantha) [inline src/App.js]: - [ ] Please create a <`Navbar>` component, which has 2 `<Link>`s. One to `/` and one to `/categories` as per requirement 👍

File: src/components/catagories.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/button-has-type */
```
Comment (codepantha) [inline src/components/catagories.js]: - [ ] Please don't disable linters. They are there to help you adhere to strict guidelines 😎

File: src/components/catagories.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import React from 'react';
+import '../App.css';
+
+const Categories = () => (
+  <>
+    <div>
+      <button type="button">Check Status </button>
+    </div>
+  </>
```
Comment (codepantha) [inline src/components/catagories.js]: - [ ] According to requirements, this `<Categories>` component should contain a `<Navbar>` component, which has 2 `<Link>`s. One to `/` and one to `/categories`. Please implement this change 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Talha-Hanif5666 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ Initialized project with components 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/catagories.js (hunk 3 above)
Comment (Talha-Hanif5666) [reply src/components/catagories.js]: Hello @codepantha, according to instructions category component contains only the check status button. Please correct me if I am wrong.

**Review by danmainah — APPROVED** [review]
Hi @Talha-Hanif5666 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**